### DISCUSSION TITLE
We need mother file for translation like translation_en_EN, US

### DISCUSSION THREAD

**Discussion by ghost** [jamulussoftware/jamulus#2679] [Ideas]
As a Korean translator. 
I am trying to translate into Korean bring it from the file 'translation_fr_FR.ts' first. 

as for the translator like me, I think there should be one mother file like 'translation_en_EN.ts' or 'translation_en_US.ts' for keeping default translation status.

and further more, I think that it would be good for us, If it is possible which supporting for the good environment for the translators.

In my experiences, I was contributing, not much these days though,  a lot of project for open source community as a translator especially for Linux Desktop Environments like MATE, Xfce, Cinnamon, Pantheon, LXQT...

During contributing,  I have met really good tools like transifex, weblate, launchped. 
those are very efficient to manage translation with good quality and easy addressing for new translators.

I hope, Jamulus team could consider good tools to translate.
Thank you.

**Comment by ignotus666** [comment]
Hi, I actually started a discussion about using Weblate [here](https://github.com/jamulussoftware/jamulus/discussions/2675#discussion-4173832). We'll see if it ends up being accepted/implemented.

Unless you are comparing a .ts file cloned some time ago with another cloned more recently, I can't see how they could have different source strings - they all get updated at the same time and should contain the same source text. In fact, if you look [here](https://hosted.weblate.org/projects/ignotus/app-text/) (don't translate anything there - it's a test), you can see how the source text stats are exactly the same for all language files: 604 strings and 5011 words..

**Reply by ghost** [reply]
Good to know it:) 
Thanks.

**Comment by ghost** [comment]
elementary OS team, involved me, use the weblate for translation tool with github as well. They know it so well. Anyway, I give you my thumbs up to your proposal. Good luck to you and all.
